src/style.css

```diff
@@ -0,0 +1,33 @@
+.body {
+  margin: 0;
+  padding: 0;
+  background-color: #ddd;
+}
+
+.container {
+  background-color: #fff;
+  padding-bottom: 4px;
+}
+
+.stars {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
```

Comment:
Change the selector `.stars--4 :nth-child(-n + 4)` to `.stars--4 .stars__star:nth-child(-n + 4)` to target the correct child elements.